.github/workflows/sdk-spec-test.yml

```diff
@@ -1,9 +1,9 @@
 name: SDK Spec Tests
 run-name: SDK Spec Tests (${{ inputs.repo || github.repository }}/${{inputs.ref || github.ref}})
 on:
-  release:
-    types:
-      - published # runs only unstable targets
+  # release: #TODO: uncomment to enable unstable targets, at this moment there are none
+  # types:
+  # - published # runs only unstable targets
```

Comment:
do we need to keep this?

Reply:
yes, at least until we'll add gcp to be stable